data/state1/ACCESSCONTROL-ROLEACTIONS/roleactions.json

```diff
@@ -85,6 +85,900 @@
       "actionid": 13,
       "actioncode": "",
       "tenantId": "state1"
+    },
+        {
```

Comment:
@hariomsinha-eGov , @nipunarora-eGov  @prachirane-eGov 
check the role actions json for your specifiv module
see any additional action given for any roles and approve this